Database/Snowflake/Execute query.ipynb

```diff
@@ -1,206 +0,0 @@
-{
```

Comment:
please don't use space in file naming, use `_` instead

Reply:
ok